mentorship ios/Views/Relation/Relation.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  Relation.swift
+//  Created on 30/06/20
+//  Created for AnitaB.org Mentorship-iOS 
+//
+
+import SwiftUI
+
+struct Relation: View {
+    //sample data
+    @ObservedObject var relationViewModel = RelationViewModel()
+    @State var showAlert = false
+    
+    var endDate: Date {
+        return Date(timeIntervalSince1970: relationViewModel.currentRelation.endDate ?? 0)
+    }
+    
+    var body: some View {
+        NavigationView {
+            ZStack {
+                Form {
+                    //Top detail view
+                    VStack {
+                        //mentor/mentee name and end date
+                        HStack {
+                            Text(relationViewModel.personName).font(.title).fontWeight(.heavy)
```

Comment:
Please also add, if he/she is a mentor or mentee in that relationship

Reply:
okay